torch/multiprocessing/reductions.py

```diff
@@ -75,17 +75,16 @@ def reduce_event(event):
 
 
 def rebuild_tensor(cls, storage, metadata):
-    storage_offset, size, stride, requires_grad, backward_hooks = metadata
+    storage_offset, size, stride, requires_grad = metadata
     t = torch._utils._rebuild_tensor(storage, storage_offset, size, stride)
     if cls == torch.nn.parameter.Parameter:
         t = torch.nn.parameter.Parameter(t)
     t.requires_grad = requires_grad
```

Comment:
Can we warn/throw on having hooks?

Reply:
I don't think that's a good idea. For example, DistributedDataParallel sets allreduce hooks on tensors and then sends them over the network. Don't want to warn here.

Reply:
Could you add a note/warning on torch.multiprocessing doc page saying that hooks will be lost?